types/node/fs.d.ts

```diff
@@ -3614,19 +3614,32 @@ declare module 'fs' {
     export function accessSync(path: PathLike, mode?: number): void;
     interface StreamOptions {
         flags?: string | undefined;
-        encoding?: BufferEncoding | undefined;
-        fd?: number | promises.FileHandle | undefined;
+        encoding?: BufferEncoding | null | undefined;
```

Comment:
I'm a little worried about adding `null` here just from the POV that this options bag type is shared between read and write, and only one of them claims to accept `null`; is it definitely accurate? I guess probably it "just works"?

Reply:
Yeah, `null` is default only for `createReadStream`. I will revert this change